Rakefile

```diff
@@ -1,14 +1,8 @@
 # frozen_string_literal: true
 
-require 'rake/testtask'
+require 'bundler/gem_tasks'
 require 'rdoc/task'
```

Comment:
Yes, it is needed, because it's configuring Rdoc to only include specific files, otherwise `LICENSE.md`, `Gemfile` etc. needlessly show up in the generated docs.

Which version of Ruby are you using, `rdoc/task` should be in the standard library: https://docs.ruby-lang.org/en/2.5.0/RDocTask.html

Reply:
ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-linux]

Reply:
Hm, `ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-darwin17]` here, so same just on a Mac. 

Does `ruby -e 'require "rdoc/task"'` raise an exception for you?

Reply:
Thanks for your help. With your prompting, I think I've resolved this :tada:

TL;DR:

Please add the following to `methodfinder.gemspec`:

```ruby
spec.add_development_dependency 'rdoc', '~> 6.0'
```

Longer TL;DR:

* on my (Arch Linux) system, `rdoc/task` is provided by a [packaged gem](https://www.archlinux.org/packages/community/any/ruby-rdoc/), not the standard library
* `bundler exec` runs commands in a locked-down environment (`$LOAD_PATH`) which only includes standard library files and explicitly-installed project dependencies
* because `rdoc/task` is neither in the standard library (on my system) nor (currently) provided by a dependency, it is not picked up
* adding the [rdoc gem](https://rubygems.org/gems/rdoc) as a development dependency solves this problem on my system (and shouldn't hurt on systems which already have `rdoc/task.rb` in the standard library)

---

> Does `ruby -e 'require "rdoc/task"'` raise an exception for you?

No, that works fine for me outside of Bundler. However:

```
$ bundler exec ruby -e 'require "rdoc/task"'

Traceback (most recent call last):
	1: from -e:1:in `<main>'
-e:1:in `require': cannot load such file -- rdoc/task (LoadError)
```

Note: on my (Arch Linux) system, the RDoc task file resolves to `/usr/lib/ruby/gems/2.5.0/gems/rdoc-6.0.3/lib/rdoc/task.rb`, which is not a core ruby file (the [ruby](https://www.archlinux.org/packages/extra/x86_64/ruby/) package only provides `/usr/lib/ruby/2.5.0/rdoc.rb`). It's provided by an optional package, [ruby-rdoc](https://www.archlinux.org/packages/community/any/ruby-rdoc/), which I (recently) installed manually. If I uninstall that package, the error occurs outside of Bundler:

```
$ ruby -e 'require "rdoc/task"'

Traceback (most recent call last):
	2: from -e:1:in `<main>'
	1: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
/usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require': cannot load such file -- rdoc/task (LoadError)
```

I tried to rectify this yesterday by adding rdoc as a dev dependency, but Bundler appeared to ignore/skip its installation. But I've just tried it now and it appears to work:

```
$ bundler add rdoc

Fetching rdoc 6.0.3
Installing rdoc 6.0.3

$ bundler exec rake

# Running:
...
11 runs, 27 assertions, 0 failures, 0 errors, 0 skips
```

Reply:
Footnote (in case anyone else encounters this issue):

> I tried to rectify this yesterday by adding rdoc as a dev dependency, but Bundler appeared to ignore/skip its installation.

This command:

```
$ bundler add rdoc
```

\- ignores/doesn't install rdoc for me on the (pre-bundlerized) master branch:

```
Fetching gem metadata from https://rubygems.org/
Using rake 12.3.1
Using bundler 1.16.1
Using minitest 5.11.3
```

This appears to be because it's already specified as a (Rubinius) dependency.

```ruby
group :development do
  gem 'rdoc', platform: :rbx
end
```

It works if those lines are commented out:

```
$ bundler add rdoc

Fetching gem metadata from https://rubygems.org/
Resolving dependencies...
Using rake 12.3.1
Using bundler 1.16.1
Using minitest 5.11.3
Fetching rdoc 6.0.3
Installing rdoc 6.0.3
```